client/src/pages/home/home.jsx

```diff
@@ -8,12 +8,14 @@ import { getProduct } from '../../state/product/saga';
 import { getCollections } from '../../state/collections/saga';
 import { getProducts } from '../../state/productCollection/saga';
 
+import { Button } from '../../components/button/Button'
 import { BuyButton } from '../../components/buyButton/buyButton';
 
 const PageHome = () => {
   const { collections } = useSelector(collectionsSelector);
   const { products } = useSelector(productsSelector);
   const { product } = useSelector(productSelector);
+  const [ click, setClick ] = useState(false);
```

Comment:
You shouldn't need an external variable like this. Implement this functionality within the component and just pass in an array for the text variables. Also I don't think we need to have multiple text options. You could just make the prop take an a single string.